org.civicrm.reqangular/test/services/api/option-group.spec.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/angularMocks',
+  'common/models/option-group',
+  'common/mocks/services/api/option-group-mock'
+], function () {
+  'use strict';
+
+  describe('api.option-group', function () {
+    var $q, OptionGroupAPI;
+
+    beforeEach(function () {
+      module('common.models', 'common.mocks');
+    });
+
+    beforeEach(inject([
+      'api.optionGroup',
+      function (_optionGroupAPI_) {
+        OptionGroupAPI = _optionGroupAPI_;
+      }
+    ]));
+
+    beforeEach(inject(function (_$q_) {
```

Comment:
you can do this in the same `beforeEach` above, no need to have two of them

Reply:
The one above injects by names, I don't think that will work.